api/handlers/v1/resource.go

```diff
@@ -1,15 +1,91 @@
 package handlersv1
 
 import (
-	"github.com/odpf/entropy/service"
+	"context"
+	"errors"
+	"github.com/odpf/entropy/domain"
+	"github.com/odpf/entropy/pkg/resource"
+	"github.com/odpf/entropy/store"
+	entropy "go.buf.build/odpf/gwv/whoabhisheksah/proton/odpf/entropy/v1beta1"
```

Comment:
is it a personal buf package?

Reply:
We were waiting for the proton PR to get merged so we can start using from odpf/proton buf repo.